src/TokenParser/TransTokenParser.php

```diff
@@ -57,18 +57,18 @@ protected function preParse(Token $token): array
         /* If we aren't closing the block, do we have a domain? */
         if ($stream->test(Token::NAME_TYPE)) {
             $stream->expect(Token::NAME_TYPE, 'from');
-            $domain = $this->parser->getExpressionParser()->parseExpression();
+            $domain = $this->parser->parseExpression();
```

Comment:
Looks like this method is only available in Twig 3.21+.

```suggestion
            $domain = method_exists($this->parser, 'parseExpression')
                ? $this->parser->parseExpression()
                : $this->parser->getExpressionParser()->parseExpression();
```